packages/components/src/button/src/Button.css

```diff
@@ -50,7 +60,7 @@
 }
 
 /* TEXT */
-.o-ui-button-text {
+.o-ui-button .o-ui-button-text {
```

Comment:
The CSS codebase should contains none of these specifity hacks.

It only gets the maintenance of the CSS harder.

Reply:
Well, if you do this, then the Text component wouldn't follow Orbit type scale anymore

Reply:
Maybe the text component could somehow use a [:where selector](https://developer.mozilla.org/en-US/docs/Web/CSS/:where) ?

Reply:
@patricklafrance this crossed my mind, this could be a graceful solution